v2/pkg/assetserver/webview/responsewriter_darwin.go

```diff
@@ -98,16 +98,31 @@ func (rw *responseWriter) Write(buf []byte) (int, error) {
 
 	rw.WriteHeader(http.StatusOK)
 
-	var content unsafe.Pointer
 	var contentLen int
 	if buf != nil {
-		content = unsafe.Pointer(&buf[0])
 		contentLen = len(buf)
 	}
 
-	if !C.URLSchemeTaskDidReceiveData(rw.r.task, content, C.int(contentLen)) {
-		return 0, errRequestStopped
+	if contentLen > 0 {
+		// Create a C array to hold the data
+		cBuf := C.malloc(C.size_t(contentLen))
+		if cBuf == nil {
+		    return 0, fmt.Errorf("memory allocation failed for %d bytes", contentLen)
```

Comment:
`fmt` import is missing so compile fails. Fixed this in my fork: https://github.com/wailsapp/wails/compare/v2-bugfix/3550-macos-cgo-error...makew0rld:wails:patch-1